common/stats/status_server.cpp

```diff
@@ -92,6 +98,12 @@ StatusServer::StatusServer(uint16_t port, EndPointToOPMap op_map)
       ret += itor->second(nullptr);
     }
 
+    for (const auto& endpoint_name : extra_stats_endpoints_) {
```

Comment:
did you test if ostrich can handle the output like "metric..., counters..., metrics..., counters..., " ?